frontend/templates/frontend/profile/profile_edit.html

```diff
@@ -8,7 +8,7 @@
 {% block content %}
     <h1>{% trans "Edit your profile" %}</h1>
 
-    <form method="POST" class="post-form" class="post-form">{% csrf_token %}
+    <form method="POST" class="post-form" enctype=multipart/form-data>{% csrf_token %}
```

Comment:
csrf_token vllt. in nächste Zeile, um konsistent gegenüber anderen forms wie z.B. in addcontent.html zu bleiben